app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -477,6 +484,35 @@ def parse_stack_resource(resource, group)
         return uid, new_result
       end
 
+      def parse_managed_image(image)
+        uid = image.id.downcase
+
+        resource_group = ResourceGroup.where(:name => image.resource_group.downcase).find do |rg|
```

Comment:
Ah, @bronaghs  is right, we should use @data_index and then make sure this relations is stored in save_inventory.

Also this creates n+1 queries, so if @data_index is not enough, we should use different hash  and then do O(1) lookup. We can get rid of the include? right?